stories/packages/AlertDialogStories/index.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {ReactElement, useState} from 'react';
+import Modal from 'react-native-modalbox';
+import {Button, Text} from 'react-native';
+
+import AlertDialog from '../../../packages/AlertDialog/lib';
+import {ContainerDeco} from '../../../storybook/decorators';
+import {storiesOf} from '@storybook/react-native';
+import styled from '@emotion/native';
+
```

Comment:
<img width="211" alt="Screen Shot 2021-08-29 at 11 29 47 PM" src="https://user-images.githubusercontent.com/27461460/131254629-146fd0e7-3979-4c06-a522-cb04dc4bc96c.png">

I think we can make a little bit better by giving some paddings or margins :)